<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-GA1O7MP" failures="1" tests="2" name="Default test(failed)(failed)" time="1.148" errors="0" timestamp="2022-03-23T04:45:56 IST">
  <testcase classname="failedXML_Study.Failed_XML1" name="Method1" time="1.036"/>
  <testcase classname="failedXML_Study.Failed_XML1" name="Method3" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failedXML_Study.Failed_XML1.Method3(Failed_XML1.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Method3 -->
  <testcase classname="failedXML_Study.Failed_XML1" name="Method2" time="0.0">
    <ignored/>
  </testcase> <!-- Method2 -->
  <testcase classname="failedXML_Study.Failed_XML1" name="Method4" time="0.0">
    <ignored/>
  </testcase> <!-- Method4 -->
  <testcase classname="failedXML_Study.Failed_XML1" name="Method5" time="0.0">
    <ignored/>
  </testcase> <!-- Method5 -->
</testsuite> <!-- Default test(failed)(failed) -->
